content/docs/started/k8s/kfctl-istio-dex.md

```diff
@@ -78,11 +77,15 @@ Notes:
 
 * **${CONFIG_URI}** - The GitHub address of the configuration YAML file that
   you want to use to deploy Kubeflow. The URI used in this guide is
-  {{% config-uri-existing-arrikto %}}.
+  {{% config-uri-istio-dex %}}.
   When you run `kfctl apply` or `kfctl build` (see the next step), kfctl creates
   a local version of the configuration YAML file which you can further
   customize if necessary.
 
+* **Disabling Istio Installation** - In the configuration file {{% config-file-istio-dex %}} you may choose to not install Istio by removing the applications `istio-crds` and `istio-install`, if your Kubernetes cluster has an existing Istio installation.
+
+* **Default Static File Password** - The configuration file {{% config-file-istio-dex %}} contains a default [staticPasswords](https://github.com/dexidp/dex/blob/0f8c4db9f61476a8f80e60f5950992149a1cc0cb/examples/config-dev.yaml#L91-L95) user with email set to `admin@kubeflow.org` and password `12341234`. It is encouraged to change this configuration or replace it with a [Dex connector](https://github.com/dexidp/dex#connectors).
```

Comment:
Remove capital letters and clarify the heading: `Default password in static file`.

Also for clarity about who should take action, change `It is encouraged to change...` to `You should change...`.